src/cavendish_particle_tracks/_widget.py

```diff
@@ -40,9 +41,18 @@
 class ParticleTracksWidget(QWidget):
     """Widget containing a simple table of points and track radii per image."""
 
-    def __init__(self, napari_viewer: napari.viewer.Viewer):
+    @property
+    def layer_measurements(self):
+        return self._layer_measurements
+
+    @layer_measurements.setter
+    def layer_measurements(self, layer):
+        self._layer_measurements = layer
```

Comment:
```suggestion
    layer_measurements: WhateverTypeThisShouldBe
```
Are you going to add more to the getter and setter functions?


If not, youy could just delete the underscore in the member variable? You can always come back and add decorators later if you do end up needing something more complicated in the getter/setter.

Reply:
my C background is showing...